ios/FluentUI/Navigation/Shy Header/ShyHeaderView.swift

```diff
@@ -135,13 +141,35 @@ class ShyHeaderView: UIView, TokenizedControlInternal {
         }
     }
 
-    var maxHeight: CGFloat {
+    var wideAccessoryView: UIView? {
+        willSet {
+            wideAccessoryView?.removeFromSuperview()
+        }
+        didSet {
+            if let newContentView = wideAccessoryView {
+                wideContentStackView.addArrangedSubview(newContentView)
```

Comment:
I might be misunderstanding but does this mean every time we set `wideAccessoryView`, it just gets added to the content stack view?

Reply:
Yes, every time we set `wideAccessoryView `, the old one will first get removed from `wideContentStackView`, and then the new one gets added to it, similar to how a`ccessoryView` is handled.